### ISSUE TITLE
SmartCity-Accessibility Server Side #458 failed

### ISSUE THREAD

**Issue by JuanJenkinsBot** [TechnionYP5777/SmartCity-Accessibility#298]
Build 'SmartCity-Accessibility Server Side' is failing!

Failed Tests :
```

```

Changes since last successful build:
- [yaelAmitay] de0e644aab0ea2261ab5852c21d95d4996407c2f - add mapview to userInfo after login #284

[View full output](http://kapalex.ddns.net:8080/job/SmartCity-Accessibility%20Server%20Side/458/)

**Comment by JuanJenkinsBot** [comment]
Build was fixed!
